.github/workflows/main.yml

```diff
@@ -0,0 +1,81 @@
+name: Rust CI
+on:
+  push:
+  pull_request:
+    types: [opened, repoened, synchronize]
+
+jobs:
+  test:
+    name: Test Rust ${{matrix.toolchain}} on ${{matrix.os}}
+    runs-on: ${{matrix.os}}-latest
+    strategy:
+      fail-fast: false
+      matrix:
+        toolchain: [stable, nightly]
+        os: [ubuntu] # linux-only
+    steps:
+      - uses: actions/checkout@master
+        with:
+          submodules: 'true'
+      - name: Get latest CMake
+        uses: lukka/get-cmake@latest
+      - name: configure aeronmd
+        run: mkdir build && cd build && cmake -G "CodeBlocks - Unix Makefiles" ../
+        working-directory: aeron
+      - name: build aeronmd
+        run: make aeronmd && mkdir -p $(systemd-path user-binaries) && mv ./binaries/aeronmd $(systemd-path user-binaries)
+        working-directory: aeron/build
+      - name: Set environment variables
+        shell: bash
+        run: |
+           echo "$(systemd-path user-binaries)" >> $GITHUB_PATH
+      - name: Install rust
+        uses: actions-rs/toolchain@v1
+        with:
+          toolchain: ${{matrix.toolchain}}
+          override: true
+      - name: Test
+        uses: actions-rs/cargo@v1
+        with:
+          command: test
+          args: --all-targets -- --test-threads=1
+        env:
+          RUST_LOG: 'trace'
+
+  clippy:
+    name: Clippy
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@master
+      - name: Install minimal nightly with clippy and rustfmt
+        uses: actions-rs/toolchain@v1
+        with:
+          profile: minimal
+          toolchain: nightly
+          components: clippy
+          override: true
+
+      - name: Clippy
+        uses: actions-rs/cargo@v1
+        with:
+          command: clippy
+          args: --all --all-targets --all-features -- -D clippy::all -D warnings
+
+  rustfmt:
+    name: rustfmt
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@master
+      - name: Install minimal nightly with clipy
+        uses: actions-rs/toolchain@v1
+        with:
+          profile: minimal
+          toolchain: nightly
```

Comment:
Hi! Thanks for your contribution!
Is it possible to use GItHub actions with stabe Rust?

Reply:
Sure! In fact, the *tests* job (starting line 8) is run with *both* stable and nightly (see the `${{matrix.toolchain}}`variable). This particular line only concerns the job running `clippy`. We can put `stable`, `nightly`, or anything else there. I'll add a commit switching this `clippy` run to stable, hoping that addresses your concern.